mongofiles/options_test.go

```diff
@@ -215,11 +229,11 @@ func TestPositionalArgumentParsing(t *testing.T) {
 				},
 			},
 			{
-				InputArgs: []string{"mongodb://foo", "get", "foo"},
```

Comment:
any reason this test was modified? I think we can just leave it as is.

Reply:
I can leave it as it is -- sorry about that.